ot-node.js

```diff
@@ -27,9 +27,7 @@ class OTNode {
         this.logger.info('======================================================');
         this.logger.info(`             OriginTrail Node v${pjson.version}`);
         this.logger.info('======================================================');
-        this.logger.info(`Node is running in ${process.env.NODE_ENV &&
-        ['development', 'testnet', 'mainnet'].indexOf(process.env.NODE_ENV) >= 0 ?
-            process.env.NODE_ENV : 'development'} environment`);
+        this.logger.info(`Node is running in ${process.env.NODE_ENV} environment`);
```

Comment:
Are you sure that process.env.NODE_ENV is defined here?

Reply:
Fine, it is defined in initConfig function

Reply:
Yes, it is defined there